circuits/ts/__tests__/MessageValidator.test.ts

```diff
@@ -1,22 +1,60 @@
 import { expect } from "chai";
-import tester from "circom_tester";
-import { stringifyBigInts, genRandomSalt } from "maci-crypto";
+import { WitnessTester } from "circomkit";
+import { SignalValueType } from "circomkit/dist/types/circuit";
+import { genRandomSalt } from "maci-crypto";
 import { PCommand, Keypair } from "maci-domainobjs";
 
-import path from "path";
-
-import type { CircuitInputs } from "maci-core";
-
-import { getSignal } from "./utils/utils";
+import { getSignal, circomkit } from "./utils/utils";
 
 describe("MessageValidator circuit", function test() {
   this.timeout(90000);
-  let circuitInputs: CircuitInputs;
-  let circuit: tester.WasmTester;
+
+  interface ICircuitInputsType {
+    stateTreeIndex: SignalValueType;
+    numSignUps: SignalValueType;
+    voteOptionIndex: SignalValueType;
+    maxVoteOptions: SignalValueType;
+    originalNonce: SignalValueType;
+    nonce: SignalValueType;
+    cmd: SignalValueType;
+    pubKey: SignalValueType;
+    sigR8: SignalValueType;
+    sigS: SignalValueType;
+    currentVoiceCreditBalance: SignalValueType;
+    currentVotesForOption: SignalValueType;
+    voteWeight: SignalValueType;
+    slTimestamp: SignalValueType;
+    pollEndTimestamp: SignalValueType;
+  }
```

Comment:
should we move this interface to **types.ts** file? since I saw the other interfaces were moved there.